Ellen Homework 1.py

```diff
@@ -0,0 +1,35 @@
+import time 
+import random 
+
+
+random_creatures = ["gorgon", "vampire", "gruffalo", "zombie", "minatour"]
+
+
+def intro():
+
+    def greet_user(username):
+        print("Hello, " + username + "!")
+    

```

Comment:
The function definition of greet user needs to be above you creating the def intro() function. Generally you don't need to define a function inside another unless there are specific use cases.